src/Controls/src/Core/Platform/GestureManager/GesturePlatformManager.Windows.cs

```diff
@@ -310,24 +311,61 @@ void ClearContainerEventHandlers()
 		{
 			if (_container != null)
 			{
-				_container.DragStarting -= HandleDragStarting;
-				_container.DropCompleted -= HandleDropCompleted;
-				_container.DragOver -= HandleDragOver;
-				_container.Drop -= HandleDrop;
-				_container.Tapped -= OnTap;
-				_container.DoubleTapped -= OnTap;
-				_container.ManipulationDelta -= OnManipulationDelta;
-				_container.ManipulationStarted -= OnManipulationStarted;
-				_container.ManipulationCompleted -= OnManipulationCompleted;
-				_container.PointerPressed -= OnPointerPressed;
-				_container.PointerExited -= OnPointerExited;
-				_container.PointerReleased -= OnPointerReleased;
-				_container.PointerCanceled -= OnPointerCanceled;
-				_container.PointerEntered -= OnPgrPointerEntered;
-				_container.PointerExited -= OnPgrPointerExited;
-				_container.PointerMoved -= OnPgrPointerMoved;
-				_container.PointerPressed -= OnPgrPointerPressed;
-				_container.PointerReleased -= OnPgrPointerReleased;
+				if (_subscriptionFlags.HasFlag(SubscriptionFlags.ContainerDragEventsSubscribed))
```

Comment:
I think `Enum.HasFlag` has a general performance issue:
* https://learn.microsoft.com/visualstudio/profiling/performance-insights-enum-hasflag

So I normally would do:

```csharp
if ((_subscriptionFlags & SubscriptionFlags.ContainerDragEventsSubscribed) != 0)
```

That might get the perf back where it was before the enum changes.